s3torchconnector/tst/unit/test_s3writer.py

```diff
@@ -0,0 +1,65 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  // SPDX-License-Identifier: BSD
+
+import logging
+from io import BytesIO
+from typing import List, Tuple
+from unittest.mock import Mock
+
+import pytest
+from hypothesis import given
+from hypothesis.strategies import lists, binary, composite
+from s3torchconnectorclient._mountpoint_s3_client import ObjectInfo, PutObjectStream
+
+from s3torchconnector import S3Writer
+
+logging.basicConfig(
+    format="%(levelname)s %(name)s %(asctime)-15s %(filename)s:%(lineno)d %(message)s"
+)
+logging.getLogger().setLevel(1)
+
+log = logging.getLogger(__name__)
+
+TEST_BUCKET = "test-bucket"
+TEST_KEY = "test-key"
```

Comment:
Nit: Could you please remove `TEST_BUCKET` and `TEST_KEY` as their are not used anywhere, other then that I'm happy to approve.